{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/{{cookiecutter.package_name}}.py

```diff
@@ -69,3 +77,26 @@ def workbench_scenarios():
                 </vertical_demo>
              """),
         ]
+
+    @staticmethod
+    def _get_statici18n_js_url():
+        """
+        Returns the Javascript translation file for the currently selected language, if any.
+        Defaults to English if available.
+        """
+        lang_code = translation.get_language()
+        if not lang_code:
+            return
+        text_js = 'public/js/translations/{lang_code}/text.js'
+        country_code = lang_code.split('-')[0]
```

Comment:
The logic is correct but the variable name is misleading here. As country code can't be a locale.

`lang_code, _country_code = lang_code.split('-')`

Reply:
Are we sure we'll always get a [language]-[country] here?